microprofile/WFLY_13129_Microprofile_CLI_Script_Example.adoc

```diff
@@ -0,0 +1,95 @@
+= Example CLI script to evolve standalone configurations with microprofile
+:author:            JF Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+We are proposing here to define a new script in docs/examples/enable-microprofile.cli
+The script is an “example” script to be applied on default standalone configurations. 
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13129
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+
+[ ] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* WildFly
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* The script is packaged in the galleon-pack maven module. It is a default package installed when WildFly is installed.
+* Installation path is _<JBOSS_HOME>/docs/examples/enable-microprofile.cli_.
+* To launch the script: _bin/jboss-cli.sh --file=docs/examples/enable-microprofile.cli_
+* By default it applies to standalone.xml, use _-Dconfig=<standalone-full.xml|standalone-ha.xml|standalone-full-ha.xml>_
```

Comment:
Personally I am not a fan of this approach where a script updates multiple configurations, instead it would feel safer if the user had already started their chosen config and then apply the update to their config but this is a pattern we have had elsewhere.

Reply:
@darranl , just to clarify the script doesn't update multiple configurations during its execution. It updates a single configuration. By default it is standalone.xml but a user can choose to update another one (-Dconfig=<config to update>).

Reply:
Ah that is much nicer

Reply:
Thank-you, will update the description.